app/src/main/java/kr/co/marubook/presentation/moim/UserMoimFragment.kt

```diff
@@ -35,18 +38,23 @@ class UserMoimFragment : BindingFragment<FragmentUserMoimBinding>(R.layout.fragm
         binding.bindViewModel = viewModel
         binding.lifecycleOwner = viewLifecycleOwner
         userMoimAdapter = UserMoimViewPagerAdapter(
-            onPageClick = {
-                onPageClick(it)
+            onMoimClick = {
+                onMoimPageClick(it)
```

Comment:
```suggestion
     UserMoimViewPagerAdapter { onMoimPageClick(it) }
```

이 부분 이렇게 수정할 수 있지 않을까유?